yang/frr-interface.yang

```diff
@@ -48,15 +69,49 @@ module frr-interface {
         description
           "Interface description.";
       }
-    }
-  }
 
-  typedef interface-ref {
-    type leafref {
-      require-instance false;
-      path "/frr-interface:lib/frr-interface:interface/frr-interface:name";
+      leaf if-index {
+        type int32 {
+          range "0..2147483647";
+        }
+        config false;
```

Comment:
If we have state container and marked as config false then leaves under them individually need not be marked config false.

Reply:
Since this is base model (similar to ietf interface module) where operational fields are marked individually as `read only`.

Reply:
We discussed about open config and IETF approach. Open config has cleaner way of doing this by putting all operational under state container and marking them as read-only. There is anothe way Luo suggested to have operational model and config model under separate tree. We can conclude in meeting today.

Reply:
Based on the discussion, we have conclude to use separate tree for configuration and state data (under state container). I have used the separate tree for interface model.

Reply:
Based on the discussion, we have conclude to use separate tree for configuration and state data (under state container). I have used the separate tree for interface model.